docs/cmd/kn_broker_create.md

```diff
@@ -0,0 +1,42 @@
+## kn broker create
+
+Create a broker.
+
+### Synopsis
+
+Create a broker.
+
+```
+kn broker create NAME
```

Comment:
As there is a good default for a broker (named 'default') I wonder whether we can make the name optional for all broker commands.

That would help in creating a default broker like the injection would do

```
kn broker create
Broker 'default' is created
```

Reply:
I'd go for requiring the name explicitly as arg for `create`. Keeping `create` formula consistent with other resources.

Reply:
I've decided to keep `create` consistent with the rest of resource creation flow, but I've had those thoughts myself that "default" might be worth. However, we can consider "default" to be exclusive for annotation created broker, therefore keeping `name` argument required here.